spec/reference/service-specification.mdx

```diff
@@ -125,7 +125,7 @@ The following type descriptions are supported:
 * `i8`, `i16`, `i32` - similar, but signed
 * `b` - buffer until the end of input (has to be last)
 * `s` - similar, but utf-8 encoded string
-* `z` - NUL-terminated utf-8 string
+* `z` - NULL-terminated utf-8 string
```

Comment:
```suggestion
* `z` - `\0`-terminated utf-8 string
```

NUL is name for ASCII 0 character https://en.wikipedia.org/wiki/ASCII#Control_code_chart

It's probably not very clear, so maybe using the C notation will be easier.

Reply:
oops... i'll keep it.